modules/schema/src/category/gis.ts

```diff
@@ -8,6 +9,50 @@ export type {GeoJSON, Feature, Geometry, Position, GeoJsonProperties} from 'geoj
 // eslint-disable-next-line import/no-unresolved
 export type {Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon} from 'geojson';
 
+// Aggregate information for converting GeoJSON into other formats
+export type GeojsonGeometryInfo = {
+  coordLength: number;
+  pointPositionsCount: number;
+  pointFeaturesCount: number;
+  linePositionsCount: number;
+  linePathsCount: number;
+  lineFeaturesCount: number;
+  polygonPositionsCount: number;
+  polygonObjectsCount: number;
+  polygonRingsCount: number;
+  polygonFeaturesCount: number;
+};
+
+// FLAT GEOJSON FORMAT GEOMETRY
+export type FlatGeometryType = 'Point' | 'LineString' | 'Polygon';
+type RemoveCoordinatesField<Type> = {
+  [Property in keyof Type as Exclude<Property, 'coordinates'>]: Type[Property];
+};
+
+export type FlatPoint = RemoveCoordinatesField<Point> & {
+  data: number[];
+  lines: number[];
+};
+
+export type FlatLineString = RemoveCoordinatesField<LineString> & {
+  data: number[];
+  lines: number[];
+};
+
+export type FlatPolygon = RemoveCoordinatesField<Polygon> & {
+  data: number[];
+  lines: number[][];
+  areas: number[][];
+};
```

Comment:
If these types are a public API I think a jsdoc comment could be helpful on these. Might there be a more descriptive name than `lines` to use for `FlatPoint`? Without looking at the code I wouldn't have an idea what that could mean for `FlatPoint`.

Reply:
Good point, I will switch to `indices`